docs/standard/numerics.md

```diff
@@ -33,44 +33,58 @@ The .NET Framework supports the standard numeric integral and floating-point pri
   
-## SIMD-enabled vector types  
- The <xref:System.Numerics> namespace includes a set of SIMD-enabled vector types for the .NET Framework. SIMD (Single Instruction Multiple Data operations) allows some operations to be parallelized at the hardware level, which results in huge performance improvements in mathematical, scientific, and graphics apps that perform computations over vectors.  
+## Floating-point types
+
+.NET includes three primitive floating-point types, which are listed in the following table:
   
- The SIMD-enabled vector types in the .NET Framework include the following: .  In addition, System.Numerics.Vectors includes a Plane type and a Quaternion type.  
+|Type|Size (in bytes)|Approximate range|Precision|  
+|----------|--------|---------------------|--------------------|  
+|<xref:System.Single?displayProperty=nameWithType>|4|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|7 digits|  
+|<xref:System.Double?displayProperty=nameWithType>|8|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|15-16 digits|  
+|<xref:System.Decimal?displayProperty=nameWithType>|16|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
   
--   <xref:System.Numerics.Vector2>,  <xref:System.Numerics.Vector3>, and  <xref:System.Numerics.Vector4> types, which are 2-, 3-, and 4-dimensional vectors of type <xref:System.Single>.  
+Each floating-point type supports standard arithmetic operators. The <xref:System.Math?displayProperty=nameWithType> class provides methods for a broader set of mathematical functions. .NET Core 2.0 and later includes the <xref:System.MathF?displayProperty=nameWithType> class that provides methods which accept arguments of the <xref:System.Single> type.
+
+You can also work with the individual bits in <xref:System.Double> and <xref:System.Single> values by using the <xref:System.BitConverter?displayProperty=nameWithType> class. The <xref:System.Decimal?displayProperty=nameWithType> structure has its own methods, <xref:System.Decimal.GetBits%2A?displayProperty=nameWithType> and <xref:System.Decimal.%23ctor%28System.Int32%5B%5D%29?displayProperty=nameWithType>, for working with a decimal value's individual bits, as well as its own set of methods for performing some additional mathematical operations.
   
--   Two matrix types, <xref:System.Numerics.Matrix3x2>, which represents a 3x2 matrix; and <xref:System.Numerics.Matrix4x4>, which represents a 4x4 matrix.  
+The <xref:System.Double> and <xref:System.Single> types are intended to be used for values that by their nature are imprecise (such as the distance between two stars) and for applications in which a high degree of precision and small rounding error is not required. You should use the <xref:System.Decimal?displayProperty=nameWithType> type for cases in which greater precision is required and rounding error is undesirable.
```

Comment:
I've encountered a few people that think `System.Decimal` completely removes rounding error. I wonder if we could clarify that its internal representation merely helps to reduce rounding error instead (the System.Decimal docs page has some examples)...

Reply:
Yes. And the last sentence ("rounding error is undesirable") implies there is none. Better is something like "and rounding error must be minimized".